integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/RunUpgradeTest.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.service.smartevents.integration.tests;
+
+import org.junit.platform.suite.api.ConfigurationParameter;
+import org.junit.platform.suite.api.IncludeEngines;
+import org.junit.platform.suite.api.SelectClasspathResource;
+import org.junit.platform.suite.api.Suite;
+
+import static io.cucumber.junit.platform.engine.Constants.PLUGIN_PROPERTY_NAME;
+
+@Suite
+@IncludeEngines("cucumber")
+@SelectClasspathResource("upgrade-features")
+@ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "pretty")
```

Comment:
Can you use same plugins as other suites, to properly generate logs?

Reply:
:+1: Done.